template/hooks/common/functions.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Common functions for cloud hooks.
+
+status=0
+
+env_refresh() {
+
+  echo "Running env_refresh() for environment: $target_env"
+
+  #create drush command
+  drush_cmd="drush @$site.$target_env"
+  echo $drush_cmd
+
+  # Prep for BLT commands.
+  repo_root="/var/www/html/$site.$target_env"
+  export PATH=$repo_root/vendor/bin:$PATH
+  cd $repo_root
+
+  echo "Update starting..."
+  blt deploy:update
+  if [ $? -ne 0 ]; then
+      echo "Update errored."
+      status=1;
+  fi
+  echo "Updates complete."
+
+  # if [ "$target_env" != "prod" ]; then
```

Comment:
Yeah actually we should just rely on Phing for this... see my last comment here...